lib/danger/danger_suggester.rb

```diff
@@ -15,15 +17,24 @@ def lint
 
     private
 
-    # @todo
+    # @return [Array<Danger::Suggester::File>]
+    def files
+      ::GitDiff.from_string(`git diff`).files.map do |file|
+        ::Danger::Suggester::File.new(file)
+      end
+    end
+
     # @return [Array<Danger::Suggester::Suggestion>]
     def suggestions
-      [
-        ::Danger::Suggester::Suggestion.new(
-          line: 1,
-          path: 'lib/danger/danger_suggester.rb',
-        )
-      ]
+      files.select(&:suggestible?).flat_map do |file|
+        file.changes.select(&:suggestible?).map do |change|
+          ::Danger::Suggester::Suggestion.new(
+            content: change.added_content,
+            line: change.first_line_number,
+            path: file.a_path,
```

Comment:
```suggestion
+            path: file.a_path
```




<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>